src/DataSourceProvider.vue

```diff
@@ -91,6 +102,29 @@ export default {
     }
   },
   methods: {
+    onSelectChange: function(event) {
+      const id = parseInt(event.target.value, 10);
+      let value;
+
+      if (id === 'none') {
```

Comment:
If `id` is `parseInt(event.target.value, 10)`, it should never be `none`.

Reply:
@tonytlwu Fixed